src/components/Registration.jsx

```diff
@@ -101,6 +101,7 @@ function Registration(props) {
       </div>
       <div className="form-group">
         <label>Program:</label>
+
```

Comment:
The developer has updated the student model to include a student status field and modified the student table to display the program name instead of the program ID. Additionally, all tests have passed successfully.